client/gutenberg/extensions/simple-payments/product-placeholder.jsx

```diff
@@ -11,10 +11,7 @@ import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 import './product-placeholder.scss';
 
 export default ( { title = '', content = '', formattedPrice = '', multiple = false } ) => {
-	const assetUrl =
-		typeof window !== undefined && window.Jetpack_Block_Assets_Base_Url
-			? window.Jetpack_Block_Assets_Base_Url
-			: '/wp-content/plugins/jetpack/_inc/blocks';
+	const assetUrl = 'https://i0.wp.com/simison-atomic-site-1.blog/wp-content/uploads/2018/11/';
```

Comment:
Using CDN for a demo temporarily until https://github.com/Automattic/wp-calypso/pull/28378 lands.

Reply:
😡 

> *We should not merge with this change.*

You mean merge but not publish, or that you don't plan to merge this PR?

Reply:
It was here in case folks hurry up the call for testing for this morning already and they could use gutenpack-jn from this branch for testing, but I think we're good to wait for #28378 anyway.

I'll remove this line and we're good to merge the other changes. :-)